package.json

```diff
@@ -146,6 +149,7 @@
     "eslint-plugin-react": "^7.29.2",
     "eslint-plugin-react-hooks": "^4.3.0",
     "fs-extra": "^10.0.1",
+    "graphql": "15.8.0",
```

Comment:
why are we using a lower version?

Reply:
People have reported the codegen script not working on `16.x`, so a downgrade works.
Our codebase was referencing `14.7`, `15.8` and `16.3`, and only `15.8` works so far.